doc/devel/contrib/rust-and-c++.md

```diff
@@ -1,6 +1,6 @@
 # Rust and C++
 
-Taskwarrior has historically been a C++ project, but as part of an [ongoing effort to replace its storage backend](https://github.com/GothenburgBitFactory/taskwarrior/issues/2770) it now includes a Rust library called TaskChampion.
+Taskwarrior has historically been a C++ project, but as part of an [effort to replace its storage backend](https://github.com/GothenburgBitFactory/taskwarrior/issues/2770) it now includes a Rust library called TaskChampion.
 To develop Taskwarrior, you will need both a [C++ compiler and tools](./development.md), and a [Rust toolchain](https://www.rust-lang.org/tools/install).
 However, most tasks will only require you to be familiar with one of the two languages.
```

Comment:
```suggestion
```

I don't think these sentences say anything that isn't either false or obvious at this point.